README.md

```diff
@@ -1 +1,132 @@
+1. [Run the notebook in IBM Watson Studio](#6-run-the-notebook-in-ibm-watson-studio)
+
+### 1. Clone the repository
+
+```
+$ git clone https://github.com/IBM/prediction-using-watson-machine-learning
+$ cd prediction-using-watson-machine-learning
+```
+
+### 2. Create Watson services in IBM Cloud
+
+* Create a new project by clicking `+ New project` and choosing `Data Science`:
+
+![](https://raw.githubusercontent.com/IBM/pattern-images/master/watson-studio/project_choices.png)
+
+> Note: Services created must be in the same region, and space, as your Watson Studio service.
+> Note: If this is your first project in Watson Studio, an object storage instance will be created.
+
+* Under the `Settings` tab, scroll down to `Associated services`, click `+ Add service` and choose `Watson`:
+
+![](https://github.com/IBM/pattern-images/blob/master/watson-studio/add_service.png)
+
+* Search for `Machine Learning`, Verify this service is being created in the same space as the app in Step 1, and click `Create`.
+
+  ![](https://raw.githubusercontent.com/IBM/pattern-images/master/machine-learning/create-machine-learning.png)
+
+* Alternately, you can choose an existing Machine Learning instance and click on `Select`.
+
+  ![](https://raw.githubusercontent.com/IBM/pattern-images/master/watson-studio/watson-studio-add-existing-ML.png)
+
+* The Watson Machine Learning service is now listed as one of your `Associated Services`.
+
+* Click on the `Settings` tab for the project, scroll down to `Associated services` and click `+ Add service` ->  `Spark`.
+
+* Either choose and `Existing` Spark service, or create a `New` one
+
+  ![](https://raw.githubusercontent.com/IBM/pattern-images/master/watson-studio/add_existing_spark_service.png)
+
+  ![](https://raw.githubusercontent.com/IBM/pattern-images/master/watson-studio/add_new_spark_service.png)
+
+### 3. Save the credentials for your Watson Machine Learning Service
+
+* In a different browser tab go to [http://console.bluemix.net](http://console.bluemix.net) and log in to the Dashboard.
+
+* Click on your Watson Machine Learning instance under `Services`, click on `Service credentials` and then on `View credentials` to see the credentials.
+
+  ![](https://raw.githubusercontent.com/IBM/pattern-images/master/machine-learning/ML-service-credentials.png)
+
+* Save the username, password and instance_id to a text file on your machine. You’ll need this information later in your Jupyter notebook.
+
+### 4. Create the DB2 Warehouse on Cloud Service and load data
+
+* Create a [Db2 Warehouse on Cloud Service](https://console.bluemix.net/catalog/services/db2-warehouse-on-cloud/) instance (an entry plan is offered).
+
+* Create the `DRUG_TRAIN_DATA_UPDATED` table in Db2 Warehouse on Cloud. You will use the [drug_train_data_updated.csv](data/drug_train_data_updated.csv) file from this git repository.
+
+* Click `Open the console` to get started with Db2 Warehouse on Cloud icon.
```

Comment:
I'm confused. Maybe "click 'open the console' icon to get started with Db2 warehouse on cloud"?